src/v2/Apps/ViewingRoom/Routes/Works/Components/ViewingRoomCarousel.tsx

```diff
@@ -105,7 +110,8 @@ export const ViewingRoomCarouselFragmentContainer = createFragmentContainer(
       fragment ViewingRoomCarousel_artwork on Artwork {
         images {
           internalID
-          resized(height: 550) {
+          # requesting the largest size and resizing it down to 550*2 for retina
+          resized(height: 1100, version: "normalized") {
```

Comment:
@dzucconi, @damassi - this does seem to increase the quality of the image dramatically, but I"m not sure it's a good tradeoff size wise to use 'normalize' image here. The next size down is 'larger' image which seem to fit the box 1024 which depending on the ratio would be scaling the image up dramatically to get to 1100 to fit the retina 550*2.

You think this requesting of normalized is acceptable here?

Reply:
I think requesting normalized is fine here. Really the only time I'd imagine this being a problem with the current Gemini architecture is something like the artwork grids (scenarios where a user may encounter n-pages of large grids of long-tail images).